src/selectors.js

```diff
@@ -569,6 +556,32 @@ export const getChainsWithBalances = createSelector(
     })
 )
 
+// Returns the status of the "Port" feature on the active chain
+export const portFeatureStatus = createSelector(
+  getActiveWalletCoinBalance,
+  getActiveWalletMetBalance,
+  getChainsWithBalances,
+  getActiveChain,
+  getIsOnline,
+  getConfig,
+  // eslint-disable-next-line max-params
+  (coinBalance, metBalance, chainsBalances, activeChain, isOnline, config) =>
+    config.enabledChains.length > 0
+      ? isOnline
+        ? utils.hasFunds(coinBalance)
+          ? utils.hasFunds(metBalance)
+            ? chainsBalances.filter(
+                chain =>
+                  chain.id !== activeChain && utils.hasFunds(chain.coinBalance)
```

Comment:
Will this scale when the wallets gets support for a 3rd chain?

Reply:
Yes, this will only disable the Port button if all possible destinations have no funds.

Reply:
When we get support for a 3rd chain we will require more specific validations inside the Port form itself and check for funds after a destination chain is selected. That would need further thought and some UI changes so I think this check is enough for now.